tasks/build.js

```diff
@@ -3,6 +3,9 @@ import gulpSequence from 'gulp-sequence';
 
 gulp.task('build', gulpSequence(
   'clean', [
+    'fonts',
+    'dict'
+  ], [
```

Comment:
`font` and `dict` should be before `manifest` because `manifest` refers directories copied by these tasks.